lib/flagon/inspector.rb

```diff
@@ -17,6 +17,13 @@ def when_enabled(flag_name)
       end
     end
 
+    def ensure_flags_exist(*flags)
```

Comment:
Using a bang usually signifies that the method causes a mutation to the object the method is called upon.